edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.IoTHub/EdgeAgentConnection.cs

```diff
@@ -676,14 +720,24 @@ internal static void VerifyTwinSignatureSuceeded()
                 Log.LogInformation((int)EventIds.VerifyTwinSignatureSuceeded, "Twin Signature is verified");
             }
 
-            internal static void TwinPropertiesAreSigned()
+            internal static void ManifestSigningIsEnabled()
+            {
+                Log.LogDebug((int)EventIds.ManifestSigningIsEnabled, $"Manifest Signing is enabled");
+            }
+
+            internal static void ManifestSigningIsNotEnabled()
```

Comment:
Would it make sense to elevate some of these events?  I'd image a customer wouldn't want to need to turn on debug logs to find out the deployment they signed was not able to be checked because the manifest trust bundle was not configured.